product-service/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "product-service",
+  "version": "1.0.0",
+  "description": "Product Service created with API Gateway and Lambdas",
+  "main": "serverless.ts",
+  "scripts": {
+    "deploy:dev": "sls deploy --stage dev",
+    "deploy:prod": "sls deploy --stage prod",
+    "test": "jest --silent"
+  },
+  "engines": {
+    "node": ">=14.15.0"
+  },
+  "dependencies": {
+    "@middy/core": "^2.5.3",
+    "@middy/http-json-body-parser": "^2.5.3"
+  },
+  "devDependencies": {
+    "@serverless/typescript": "^3.0.0",
+    "@types/aws-lambda": "^8.10.71",
+    "@types/jest": "^28.1.7",
+    "@types/node": "^14.14.25",
+    "@typescript-eslint/eslint-plugin": "^5.34.0",
+    "@typescript-eslint/parser": "^5.34.0",
+    "esbuild": "^0.14.11",
+    "eslint": "^8.22.0",
+    "jest": "^28.1.3",
+    "json-schema-to-ts": "^1.5.0",
+    "serverless": "^3.0.0",
+    "serverless-auto-swagger": "^2.8.2",
+    "serverless-esbuild": "^1.23.3",
+    "serverless-webpack": "^5.8.0",
```

Comment:
tried to use both esbuild and webpack =)

Reply:
Yes, esbuild dependency is redundant here. Thanks!